mantidproject/mantid
# Add more default plot settings to workbench

This PR adds more default plot settings to workbench:
- Axes spine width
- Whether grid is turned on
- Enable/disable ticks on each side of the plot
- Enable/disable tick labels on each side of the plot
- Set width, length, and direction for major and minor ticks

**To test:**
- Ensure enabling/disabling all of the settings above work correctly.
- Are the changes to the gui clear and easy to use in your opinion? 
- Should we add some of these settings to the plot config? For example, a tiled plot with ticks and labels on all sides can look a bit messy!
![image](https://user-images.githubusercontent.com/68589504/101467209-ef8f5800-3939-11eb-9bed-9f9258d4c581.png)


Fixes #29739 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
